components/count-badge/test/count-badge.visual-diff.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+		import '../count-badge.js';
+		import '../../typography/typography.js';
+	</script>
+	<title>d2l-count-badge</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<meta charset="UTF-8">
+</head>
+<body class="d2l-typography">
```

Comment:
Might seem silly, but would probably be good to have a test or two for the `hide-zero` property.  Maybe one where the property is set but not the number isn't zero so it shows, and one where it is 0 so it doesn't show